torch/jit/_recursive.py

```diff
@@ -19,6 +19,8 @@
 
 # TODO: there should be a more principled way of doing this.
 ignored_attributes = [
+    "qconfig",
+    "_qconfig_map",
```

Comment:
@eellison As you predicted there are some attributes now getting scripted that shouldn't be. I would prefer to ignore them by modifying `__jit_ignored_attributes__` whereever they are added but I can't find the code that adds the `qconfig` attribute. I think `_qconfig_map` is added here: https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantize.py#L429